NBitcoin/IpExtensions.cs

```diff
@@ -183,7 +214,7 @@ public static byte[] GetGroup(this IPAddress address)
 				vchRet.Add((byte)(bytes[15 - 2] ^ 0xFF));
 				return vchRet.ToArray();
 			}
-			else if (address.IsTor())
+			else if (address.IsTor() || address.IsCjdns())

```

Comment:
Should the tests from https://github.com/bitcoin/bitcoin/blob/2878167c18eee9ad440ab3f663d74bf2db254f17/src/test/netbase_tests.cpp#L314 be ported too?

Reply:
I don't know. I just reused the existing method. The core's `GetGroup`, as well as other methods, use the new network maps that I still don't fully understand.